ChangeLog.md

```diff
@@ -1,4 +1,8 @@
 # Branch Android SDK change log
+- v5.11.0
+  * _*Master Release*_ - Apr 1, 2024
+  - Added support for setting Branch API URL through `branch.json`
```

Comment:
Add a mention for improved performance for best case for user agent string fetch

Reply:
Ah good call out. Just updated it.